docs/_static/custom.js

```diff
@@ -1,40 +1,43 @@
-$(document).ready(function () {
-  var sections = $('div.section');
-  var activeLink = null;
-  var bottomHeightThreshold = $(document).height() - 30;
-
-  $(window).scroll(function (event) {
-    var distanceFromTop = $(this).scrollTop();
-    var currentSection = null;
-
-    if(distanceFromTop + window.innerHeight > bottomHeightThreshold) {
-      currentSection = $(sections[sections.length - 1]);
-    }
-    else {
-      sections.each(function () {
-        var section = $(this);
-        if (section.offset().top - 1 < distanceFromTop) {
-          currentSection = section;
-        }
-      });
-    }
-
-    if (activeLink) {
-      activeLink.parent().removeClass('active');
-    }
-
-    if (currentSection) {
-      activeLink = $('.sphinxsidebar a[href="#' + currentSection.attr('id') + '"]');
-      activeLink.parent().addClass('active');
-    }
+'use-strict';
 
-  });
+let activeLink = null;
+let bottomHeightThreshold, sections, tables;
+
+document.addEventListener('DOMContentLoaded', () => {
+  bottomHeightThreshold = document.documentElement.scrollHeight - 30;
+  sections = document.querySelectorAll('div.section');
 
-  const tables = document.querySelectorAll('.py-attribute-table[data-move-to-id]');
+  tables = document.querySelectorAll('.py-attribute-table[data-move-to-id]');
```

Comment:
Since this variable isn't used anywhere, it'd be best to just leave it with the previous `const` declaration.